GameDataParser/Parsers/SkillParser.cs

```diff
@@ -44,69 +43,57 @@ public static List<SkillMetadata> Parse(MemoryMappedFile m2dFile, IEnumerable<Pa
                     }
                     XmlNode kinds = node.SelectSingleNode("kinds");
                     metadata.Type = kinds.Attributes["type"].Value != null ? int.Parse(kinds.Attributes["type"].Value) : 0;
-                    metadata.SubType = kinds.Attributes["subType"].Value != null ? int.Parse(kinds.Attributes["subType"].Value) : 0;
-                    metadata.RangeType = kinds.Attributes["rangeType"].Value != null ? int.Parse(kinds.Attributes["rangeType"].Value) : 0;
-                    metadata.Element = kinds.Attributes["element"].Value != null ? int.Parse(kinds.Attributes["element"].Value) : 0;
+                    if (kinds.Attributes.GetNamedItem("subType") != null)
+                    {
+                        // Get Rank
+                        metadata.SubType = kinds.Attributes["subType"].Value != null ? int.Parse(kinds.Attributes["subType"].Value) : 0;
```

Comment:
redundant check. also if it's null, then SubType is not being set (don't think that matters though?)

Reply:
Alright, just leaving the lambda for the check.